Sources/SecureEnclaveAccessControl.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  SecureEnclaveAccessControl.swift
+//  Valet
+//
+//  Created by Dan Federman on 9/18/17.
+//  Copyright © 2017 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+
+@objc(VALSecureEnclaveAccessControl)
+public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable {
+    /// Access to keychain elements requires user presence verification via Touch ID, Face ID, or device Passcode. Keychain elements are still accessible by Touch ID even if fingers are added or removed. Touch ID does not have to be available or enrolled.
+    /// @version Available on iOS 8 or later, and macOS 10.11 or later.
```

Comment:
`- version:`, or just remove the `@version`.  Either way, should be consistent across the cases.

Reply:
nice find :)